cpp/include/nvstrings/NVText.h

```diff
@@ -55,6 +55,23 @@ class NVText
      */
     static NVStrings* unique_tokens(NVStrings& strs, const char* delimiter=nullptr);
 
+    /**
+     * @brief Tokenize all input strings into single-character strings.
+     *
+     * Example
+     * ```
+     * s = ['apple','pear']
+     * r = character_tokenize(s)
+     * r is now ['a','p','p','l','e','p','e','a','r']
+     * ```
+     *
+     * Note these are UTF-8 characters and so each string may be one or more bytes.
```

Comment:
Each token you mean?  Just out of curious, is there a test for that?  I wasn't aware nv string handles utf-8 code points, it would be interesting to try it out. ;-)

Reply:
I don't have a C++ test that shows multi-byte UTF-8 characters but the python test does have some here: https://github.com/rapidsai/cudf/blob/fd4ee15e4f892e83507a5b1a94bb6b12dd4e0325/python/nvstrings/tests/test_text.py#L101
Hope that helps.

Reply:
Thanks, awesome!